pkg/hardwareutils/bmc/access_test.go

```diff
@@ -432,6 +432,18 @@ func TestStaticDriverInfo(t *testing.T) {
 			power:      "",
 		},
 
+		{
+			Scenario:   "redfish uefi http boot",
+			input:      "redfish-https+https://192.168.122.1",
```

Comment:
hmm, according to the design doc I should have called this redfish-uefihttp
https://github.com/metal3-io/metal3-docs/blob/main/design/baremetal-operator/uefi-http-boot.md

Reply:
Yeah, let's use that name: redfish-https+https will be confused with redfish+https for sure

Reply:
Does PR title needs update then as well?

Reply:
Yeah I was wondering the same should the title be Add "redfish-uefihttp" interface?
Or should we refer to it as  Add supportfor  "redfish-http" boot interface?